gitlab/tests/test_config.py

```diff
@@ -87,13 +93,19 @@ def test_invalid_id(self, m_open):
     @mock.patch('six.moves.builtins.open')
     def test_invalid_data(self, m_open):
         fd = six.StringIO(missing_attr_config)
-        fd.close = mock.Mock(return_value=None)
+        fd.close = mock.Mock(return_value=None,
+                             side_effect=lambda: fd.seek(0))
         m_open.return_value = fd
         config.GitlabConfigParser('one')
+        config.GitlabConfigParser('one')
```

Comment:
I don't think this duplicated line is useful :) Could you remove it?

Reply:
I added this to test the change line#97.

Without the fd.seek(0), all calls to config.GitlabConfigParser fail except the 1st one:

``` python
# fd.tell() = 0
config.GitlabConfigParser('one')  # reads fd
# fd.tell() = end of file
config.GitlabConfigParser('one')  # reads nothing because fd is at end of file, so it fail
```

Reply:
OK, thanks for the explanation.